scripts/swap-package-links.js

```diff
@@ -0,0 +1,39 @@
+const fse = require('fs-extra');
+const path = require('path');
+const {
+  displayConfirmationMessage,
+  displayErrorMessage
+} = require('./common');
+
+const [/* node bin */, /* path to this script */, pkgName] = process.argv;
+
+const PACKAGE_LOCATIONS = {
+  handsontable: './tmp',
+  '@handsontable/angular': './wrappers/angular-handsontable/dist/hot-table'
+};
+let packageList = {};
+
+if (pkgName) {
+  packageList[pkgName] = PACKAGE_LOCATIONS[pkgName];
+
+} else {
+  packageList = PACKAGE_LOCATIONS;
+}
+
+for (const [packageName, packageLocation] of Object.entries(packageList)) {
```

Comment:
This can be simplified if you use `new Map()` as a holder for package locations.
```suggestion
for (const [packageName, packageLocation] of packageList) {
```